capstone/helpers/image_processing.py

```diff
@@ -0,0 +1,111 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import skimage.io as io
+import skimage.transform as transform
+import skimage.exposure as exposure
+import glob
+import os
+
+
+def print_columns(data, cols):
+    """
+    Splits a list into sublists and prints it out in columns
+    :params data: list of data to be printed
+    :params cols: number of columns
+    :return: print the data and return none
+    """
+    num_per_col = int(np.ceil(len(data) / cols))
+    split = [data[i:i + num_per_col] for i in range(0, len(data), num_per_col)]
```

Comment:
You might use `xrange` instead of `range` here. 
`range ` creates list
`xrange` creates generator
Thus for longer sequences `xrange` saves RAM memory.